scripts/sanitycheck

```diff
@@ -3237,6 +3229,8 @@ def main():
     discards = []
     if options.load_tests:
         ts.load_from_file(options.load_tests)
+    elif options.only_failed:
+        ts.get_last_failed()
```

Comment:
OMG, this is line three thousand two hundred and thirty three.

Reply:
:)

Reply:
yeah, we need to refactor the whole thing at some point, things have been adding up.